src/slack_message_event.py

```
@@ -0,0 +1,9 @@
+class SlackMessageEvent:
+    def __init__(self, event):
+        self.member_id = event.get("event").get("user")
+        self.team_id = event.get("event").get("team")
+        self.channel = event.get("event").get("channel")
+        self.token = event.get("token")
+
+        if self.member_id == None or self.team_id == None or self.channel == None or self.token == None:
+            raise ValueError

```

Additionally, you never want to raise an exception without giving more information - how is the developer supposed to know which value was in error?
```python
raise ValueError("member_id cannot be None")
```